autogluon/training/docker/1.2/py3/cu124/Dockerfile.gpu.py_scan_allowlist.json

```diff
@@ -0,0 +1,14 @@
+{
+    "72236": "setuptools - autogluon uses setuptools == 69.5.1",
+    "72089": "NLTK - autogluon uses nltk <= 3.8.1",
+    "65189": "Ray - autogluon uses ray <= 2.39.0",
+    "71579": "mlflow - parent image uses mlflow <= 2.18.0",
+    "71587": "mlflow - parent image uses mlflow <= 2.18.0",
+    "71693": "mlflow - parent image uses mlflow <= 2.18.0",
+    "71578": "mlflow - parent image uses mlflow <= 2.18.0",
+    "71577": "mlflow - parent image uses mlflow <= 2.18.0",
+    "71692": "mlflow - parent image uses mlflow <= 2.18.0",
+    "71584": "mlflow - parent image uses mlflow <= 2.18.0",
+    "71589": "mlflow - parent image uses mlflow <= 2.18.0",
```

Comment:
we shall notify the DLC team to patch the base torch image?

Reply:
we could if we are comfortable with pushing our release to next week